### PR TITLE
Clean up 18

### PR DESCRIPTION
**Pull request by DelsFdez**
## Issue 
Spring cleaning for IWD

### CODE REVIEW

File: app/src/main/java/com/montreal/wtm/WTMApplication.kt [hunk 1]
```diff
@@ -44,7 +44,7 @@ class WTMApplication : Application() {
         .build()
     Twitter.initialize(config)
 
-    Fabric.with(this, Crashlytics());
+//    Fabric.with(this, Crashlytics());
```
Comment (ldev6) [inline app/src/main/java/com/montreal/wtm/WTMApplication.kt]: no Fabric ?

File: app/src/main/java/com/montreal/wtm/ui/activity/TalkActivity.java [hunk 2]
```diff
@@ -2,117 +2,139 @@
...
 import android.support.design.widget.CollapsingToolbarLayout;
 import android.support.design.widget.FloatingActionButton;
-import android.support.v7.app.AppCompatActivity;
+import android.support.design.widget.Snackbar;
+import android.support.v4.widget.ImageViewCompat;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
 import android.support.v7.widget.Toolbar;
-import android.text.Html;
+import android.util.Log;
```
Comment (ldev6) [inline app/src/main/java/com/montreal/wtm/ui/activity/TalkActivity.java]: Remove this import

File: app/src/main/java/com/montreal/wtm/ui/activity/TalkActivity.java [hunk 3]
```diff
@@ -126,4 +148,77 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
+
+    private void saveRating(int rating) {
+        FirebaseData.INSTANCE.saveSessionRating(talk.getSessionId(), rating);
+    }
+
+    private FirebaseData.RequestListener<Pair<String, Long>> requestSessionRatingListener =
+        new FirebaseData.RequestListener<Pair<String, Long>>() {
+            @Override
+            public void onDataChange(@Nullable Pair<String, Long> data) {
+                Log.d(TAG, "Received session rating " + data.getFirst() + ", " + data.getSecond());
```
Comment (ldev6) [inline app/src/main/java/com/montreal/wtm/ui/activity/TalkActivity.java]: to remove

File: app/src/main/java/com/montreal/wtm/ui/activity/TalkActivity.java [hunk 4]
```diff
@@ -126,4 +148,77 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
+                try {
+                    highlightStars(data.getSecond());
+                } catch (NumberFormatException ex) {
+                    Log.w(TAG, "Invalid rating", ex);
```
Comment (ldev6) [inline app/src/main/java/com/montreal/wtm/ui/activity/TalkActivity.java]: Put in fabric instead ?

File: app/src/main/java/com/montreal/wtm/ui/fragment/ProgramFragment.java [hunk 5]
```diff
@@ -143,32 +134,57 @@ public String getStartDate(String stringDate, String startTime, String endTime,
...
 
-            Date result = new Date(newDateTime);
-            int minutes = result.getMinutes();
-            minutes = minutes > 9 ? minutes : '0' + minutes;
-            Log.v("result", " result = " + result.getHours() + ":" + minutes);
-            return result.getHours() + ":" + minutes;
+            return getTime(newStartTime) + " - " + getTime(newEndTime);
         } catch (ParseException e) {
-            e.printStackTrace();
+            Log.e(TAG, "ParseException", e);
```
Comment (ldev6) [inline app/src/main/java/com/montreal/wtm/ui/fragment/ProgramFragment.java]: remove log and add Fabric log

File: app/src/main/java/com/montreal/wtm/ui/fragment/ProgramFragment.java [hunk 6]
```diff
@@ -143,32 +134,57 @@ public String getStartDate(String stringDate, String startTime, String endTime,
...
         FirebaseData.INSTANCE.getMyShedule(getActivity(), requestListenerMySchedule);
+        FirebaseData.INSTANCE.getSessions(getActivity(), requestListenerSession);
+        FirebaseData.INSTANCE.getMyRatings(getActivity(), requestListenerRatings);
     }
 
+    private FirebaseData.RequestListener<HashMap<String, Long>> requestListenerRatings =
+        new FirebaseData.RequestListener<HashMap<String, Long>>() {
+            @Override
+            public void onDataChange(@org.jetbrains.annotations.Nullable HashMap<String, Long> data) {
+                Log.d(TAG, "Received ratings:" + data);
```
Comment (ldev6) [inline app/src/main/java/com/montreal/wtm/ui/fragment/ProgramFragment.java]: remove log

**Review by ldev6 — CHANGES_REQUESTED** [review]
Small change

File: app/src/main/java/com/montreal/wtm/ui/fragment/ProgramFragment.java (hunk 6 above)
Comment (DelsFdez) [reply app/src/main/java/com/montreal/wtm/ui/fragment/ProgramFragment.java]: Won't remove the logs, they should be omitted with proguard

File: app/src/main/java/com/montreal/wtm/ui/activity/TalkActivity.java (hunk 4 above)
Comment (DelsFdez) [reply app/src/main/java/com/montreal/wtm/ui/activity/TalkActivity.java]: You mean crashlytics?

Comment (ldev6) [reply app/src/main/java/com/montreal/wtm/ui/activity/TalkActivity.java]: Yes

File: app/src/main/java/com/montreal/wtm/ui/fragment/ProgramFragment.java (hunk 6 above)
Comment (ldev6) [reply app/src/main/java/com/montreal/wtm/ui/fragment/ProgramFragment.java]: Ok

**Comment by ldev6** [comment]
Lgtm

**Pull request merged**